lib/terminal-table/style.rb

```diff
@@ -62,6 +62,15 @@ def defaults= options
           @@defaults = defaults.merge(options)
         end
       end
+
+      def on_change attr
```

Comment:
Why do we have to do it this way rather than simply changing the `width` accessor?